docs/guide/docs/nlu/intro-nlu.md

```diff
@@ -0,0 +1,32 @@
+---
+id: intro-nlu
+title: Natural Language Unit
+---
+
+Natural language understanding is a branch of AI that interprets and understands text from a user then converts the text into a usable format for computers. For example, Botpress' NLU transforms natural dialog from the user into structured information that your chatbot can understand and use. 
+
+The information generated by NLU tasks is added to the message metadata directly (under `event.nlu`), ready to be consumed by other modules and components.
+
+Below are some tasks performed by the NLU module
+![NLU Tasks](../assets/nlu-overview.png)
+
+ > **QnA**: A simple use-case for chatbots is to understand a question and provide an answer automatically. Doing that manually for all the questions and answers using the NLU module and the flow editor would be a tedious task, which is why we recommend using the QnA module for that instead.
+
+## Language Server
+The language server provides additional information about words, which allows your chatbot to understand words with similar meanings. It takes away the need for you to teach your chatbot about synonyms, colloquial language, and content translation. By default, your Botpress server will query one of our language servers for that purpose. You can also choose to host your server if you would like to keep everything on your premises. Head over to the [Hosting](../infrastructure/hosting#running-your-language-server) page for more details.
```

Comment:
Can you remove content translation. When developing a multi-lingual chatbot you need to translate your content